includes/cub.h

```diff
@@ -48,6 +49,50 @@ typedef struct s_pair_i {
 	int	y;
 }				t_pair_i;
 
+typedef struct s_window {
```

Comment:
I'm a bit confused about this window struct, why is it necessary? I mean, before we were also creating a window with mlx and it wasn't necessary

Reply:
You're right, we do not really need this. A left-over 🤷🏻‍♂️
Removing this...